internal/common/update/patch_payload.go

```diff
@@ -11,12 +12,28 @@ import (
 )
 
 type attrPatchOperations struct {
-	data map[string][]jsondiff.Operation
+	data          map[string][]jsondiff.Operation
+	ignoreInState []string
 }
 
-func newAttrPatchOperations(patch jsondiff.Patch) *attrPatchOperations {
+func (m *attrPatchOperations) ignoreInStatePath(path string) bool {
```

Comment:
Changes to this file are not necessary for this PR, but will most likely be needed later.

Reply:
If this is not necessary now, I would remove it and add it when it's needed